pkgs/applications/editors/jupyter-kernels/coq/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, callPackage
+, runCommand
+, makeWrapper
+, coq
+, imagemagick
+, python3
+}:
+
+# Jupyter console:
+# nix run --impure --expr 'with import <nixpkgs> {}; jupyter-console.withSingleKernel coq-kernel.definition'
+
```

Comment:
you put this in every one of your contribution but I am not a fan as it can get outdated pretty quickly. I think it's best to add a test in passthru.test that just tries to build the package as is done for neovim (if you want an example) and/or improve the documentation: either in the manual or in a README for jupyter that you could link here as a comment.
The `with import <nixpkgs> {};` tends to be outdated as a syntax so better not spread it.

Reply:
Tests already exist; the kernels are built as part of the `jupyter-all` derivation.

I agree this isn't the best place for documentation. As part of the effort to make the kernels more organized and homogeneous, I'd like to have nice ways of launching them which are explained in the manual. As things are still a little fragmentary, I think it's useful to keep these comments at the top explaining how to launch each particular kernel. But removing them is definitely part of my multi-step plan :)

Would you be happier with `with import ./. {};` ?

Reply:
Thanks!